packages/@react-types/calendar/src/index.d.ts

```diff
@@ -62,7 +62,12 @@ export interface CalendarPropsBase {
    * Controls the behavior of paging. Pagination either works by advancing the visible page by visibleDuration (default) or one unit of visibleDuration.
    * @default visible
    */
-  pageBehavior?: PageBehavior
+  pageBehavior?: PageBehavior,
+  /**
+   * The day that starts the week.
+   * @default 'Sun'
+   */
+  firstDayOfWeek?: 'Sun' | 'Mon' | 'Tue' | 'Wed' | 'Thu' | 'Fri' | 'Sat'
```

Comment:
Maybe we should go with lower case for these? Most of our APIs that take enums are lower case.

Also, maybe there should not be a `@default`? The real default depends on the locale so it's not always Sunday